src/Microsoft.WSMan.Management/CredSSP.cs

```diff
@@ -671,23 +654,15 @@ private void UpdateGPORegistrySettings(string applicationname, string[] delegate
                 //open the registry key.If key is not present,create a new one
                 Credential_Delegation_Key = rootKey.OpenSubKey(Registry_Path_Credentials_Delegation, true);
                 if (Credential_Delegation_Key == null)
-                    Credential_Delegation_Key = rootKey.CreateSubKey(Registry_Path_Credentials_Delegation
-#if !CORECLR
-                        , RegistryKeyPermissionCheck.ReadWriteSubTree
-#endif
-                        );
+                    Credential_Delegation_Key = rootKey.CreateSubKey(Registry_Path_Credentials_Delegation, RegistryKeyPermissionCheck.ReadWriteSubTree);
```

Comment:
Why did you decide to add RegistryKeyPermissionCheck parameter here?
It looks like the macro was removing it previously...

Reply:
I guess it was removed because CoreFX had not support. Now we have this and we can enable the check.